help/develop/domains/chem/cheminformatics.md

```diff
@@ -31,59 +31,36 @@ descriptors(table, column, descriptors);
 ```
 
 The three first functions, `searchSubstructure`, `getSimilarities` and `findSimilar`, are currently made client-based,
-using [RDKit-JS](https://github.com/rdkit/RDKitjs) library. We are planning to support both server-side and browser-side
+using [RDKit JS](https://github.com/rdkit/rdkit/blob/master/Code/MinimalLib/minilib.h). We are planning to support both server-side and browser-side
 modes for these functions in the future, while the API remains as described below.
 
```

Comment:
`We plan to support both server-side and broser-side modes. The API remains the same.`

Reply:
Done